server.go

```diff
@@ -70,6 +71,9 @@ type Server struct {
 	accountsLists  *AccountsLists
 	NodeID         string
 	AdminAccountID string
+
+	// Callback
+	OnPayloadDelivered func(slot uint64, blockHash string, parentHash string, proposerPubkey string) error
```

Comment:
wdyt of this being inside svc or outside of it?

Reply:
just for future proofing it, not all fields and potential inputs are outside of svc

Reply:
The reason I bring this up is because for OnHeaderDelivered, i have to pass in builder_extra_data
https://github.com/bloXroute-Labs/relayproxy/pull/30/files#diff-c52daedaeda258db15ec11d57bbf299412a20328f6cd026980ef07f01960dd3aR930
which isn't passed up to the calling function

Reply:
resolved on slack, we'll keep it as is for now unless @bhakiyakalimuthu you have any strong opinion on this